components/director/pkg/cert/certutils.go

```diff
@@ -68,7 +76,7 @@ func ConstructOURegex(patterns ...string) string {
 	nonEmptyStr := make([]string, 0)
 	for _, pattern := range patterns {
 		if len(pattern) > 0 {
-			nonEmptyStr = append(nonEmptyStr, pattern)
+			nonEmptyStr = append(nonEmptyStr, fmt.Sprintf("\\b%s\\b", pattern))
```

Comment:
what is the idea of `\\b` prefix/suffix?

Reply:
Match only a whole word / phrase instead of just match.

For example without it the regex "Region" will match the string "thisIsAFakeRegion", now it will match only "Region"